profile/Makefile

```diff
@@ -225,7 +230,7 @@ headers = $(source:.cpp=.h)
 
 obj := $(addprefix obj/, $(obj))
 cases := $(addprefix models/, $(cases))
-case := $(addprefix models/, $(case))
+cases += $(case)
```

Comment:
The reason I do this is that case is now a changeable variable and it could be any path(absolute or relative)+filename. Adding any prefix to it would make it no meanless.